LectureNotes/Lectures/Fall2018/Lecture4/PPP_notes.tex

```diff
@@ -0,0 +1,186 @@
+\documentclass[11pt]{article}
+%\usepackage{siunitx}
+\usepackage{lorig}
+\usepackage{dsfont}
+
+\let\oldphi\phi
+\renewcommand{\phi}{\varphi}
+
+\newenvironment{soln}{\begin{proof}[\textsc{Solution}]}{\renewcommand{\qedsymbol}{$\blacklozenge$}\end{proof}}
+
+\title{Poisson Point Processes}
+\author{Matt Lukac\thanks{\textbf{email:} mlukac@uoregon.edu}}
+\date{\today}
+\begin{document}
+\maketitle
+Recall a realization of the basic noise for Gaussian processes looked like that in Figure \ref{fig:gaussnoise}. Now, arrows are either muted or (rarely) point up. See Figure \ref{fig:poisnoise}.
+\begin{figure}
+	\centering
+	\includegraphics[width=0.9\linewidth]{gauss_noise}
+	\caption{A realization of the basic noise used to construct a Gaussian process.}
+	\label{fig:gaussnoise}
+\end{figure}
+\begin{figure}
+	\centering
+	\includegraphics[width=0.9\linewidth]{pois_noise}
+	\caption{A realization of the basic noise used to construct a Poisson process.}
+	\label{fig:poisnoise}
+\end{figure}
+
+\section{Motivation}
+Suppose in some space $X$ we lay down a large number of LED lights, each with their own battery, with density given by a $\sigma$-finite measure $\mu$. We do this in a way so that, for each region $A\subset X$, we put down about $M\mu(A)$ lights in that region, where $M$ is some large number. Independently we turn on each light with probability $M^{-1}$, and leave off otherwise.
+
+We would like to answer the following question: how many lights in $A$ are on? To that end, let $N(A)$ denote the number of lights on in $A$ and compute
+\begin{align}
+	\mathbb{E}\left[ N(A) \right] = \mathbb{E} \left[ \sum_{\text{lights in }A} \mathds{1}_{\{\text{lights on}\}} \right]

```

Comment:
the indicator should be `light on`, not `lights on`, no? since it's for a single light?